<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FVADescriptionEditorClass</class>
 <widget class="QMainWindow" name="FVADescriptionEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FVADescriptionEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbFoto">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>-50</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Редактировать поля</string>
    </property>
    <widget class="QListWidget" name="lstPeople">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QToolButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPlace">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Место</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnDel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPeople">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Люди</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBoxPlace">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Дата/др. папка</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDesc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Описание/Событие</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblComment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Коментарий/Таги</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDevice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Устройство</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbScaner">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Сканер</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDescription">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBoxScaner">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbBoxDevice">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="editComment">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="editOldName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Действие</string>
    </property>
    <widget class="QCommandLinkButton" name="btnNext">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="btnPrev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохр</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="btnDict">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>НСИ</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fvadescriptioneditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnNext</sender>
   <signal>pressed()</signal>
   <receiver>FVADescriptionEditorClass</receiver>
   <slot>onNextPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
